assembly/compliance-tool/src/main/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/exporter/SourcesExporter.java

```diff
@@ -58,6 +59,13 @@
  * </p>
  */
 class SourcesExporter {
+    /**
+     * A {@code Comparator} for sorting {@code ReleaseWithSources} objects.
+     * This comparator is used to sort the list of releases before it is
+     * written to the output CSV file.
+     */
+    static final Comparator<ReleaseWithSources> RELEASES_COMPARATOR = createReleaseComparator();
+
```

Comment:
Agreed that SourcesExporter was not the correct place for the comparator. However, as the ReleaseWithSources class is only package-local, it cannot be referenced by already existing utility classes outside the package.

I moved the class to a top-level class, so that it can be easier accessed and is no longer defined in SourcesExporter. The comparator is now defined in the exporter itself because this class should be under control of the sort order of the exported file.